gobblin-data-management/src/main/java/org/apache/gobblin/data/management/copy/iceberg/IcebergTable.java

```diff
@@ -202,4 +204,15 @@ protected void registerIcebergTable(TableMetadata srcMetadata, TableMetadata dst
       this.tableOps.commit(dstMetadata, srcMetadata.replaceProperties(dstMetadata.properties()));
     }
   }
+
+  /**
+   * Provides absolute path of the iceberg table location
+   * @return string value of table location if present
+   * @throws TableNotFoundException
+   */
+  public String getTablePath() throws TableNotFoundException {
+    Path tablePath = Paths.get(accessTableMetadata().location());
+    log.info("Iceberg table location: {}", tablePath.toAbsolutePath());
```

Comment:
Is the path returned by `accessTableMetadata()` not absolute? If not, how does this call make it absolute?